witchcraft/witchcraft.go

```diff
@@ -963,7 +962,7 @@ func (s *Server) initShutdownSignalHandler(ctx context.Context) {
 
 	go wapp.RunWithRecoveryLogging(ctx, func(ctx context.Context) {
 		sig := <-shutdownSignal
-		ctx = wparams.ContextWithSafeParam(ctx, "signal", sig.String())
```

Comment:
This context is used to control the lifecycle of `Shutdown` but is not passed to any logging. This made debugging shutdown behavior difficult because the server seemingly dies without an obvious trigger in the log stream.